### ISSUE TITLE
Swipe on Interactive search modal on mobile

### ISSUE THREAD

**Issue by ytosaas** [Radarr/Radarr#9442]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Swiping causes the movie page behind the interactive search modal to swipe to the prev/next movie and removes the search results.  The scroll bar is unaffected.

### Expected Behavior

Do not swipe the movie page behind the interactive search modal.

### Steps To Reproduce

On mobile, run an interactive search.  When results display,  swipe on approx the top 8 results to view release details or reach the grab button.

### Environment

```markdown
- OS: android
- Radarr: 5.2.3.8296
- Docker Install: no
- Using Reverse Proxy: no
- Browser: chrome
- Database: Sqlite 3.36.0
```

### What branch are you running?

Nightly

### Trace Logs? **Not Optional**

Video https://discord.com/channels/264387956343570434/264388019585286144/1176126555391533116

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by bakerboy448** [comment]
Confirmed on 5.2.3.8296 even with https://github.com/Radarr/Radarr/commit/45b38b44c1ed2108422a25770c6cc77d0f0c39d5

**Comment by bakerboy448** [comment]
Possibly fixed https://github.com/Radarr/Radarr/commit/bbffff78ed2c1a65c8050e16b0307becff996571

**Comment by ytosaas** [comment]
Confirmed fixed in nightly 5.2.3.8297. Thanks team!

**Comment by bakerboy448** [comment]
Confirmed as well
